File: 24. Stable Sort and Position/main.py
class Solution:
    # Function to get the index of an element in a sorted array
    def getIndexInSortedArray(self, arr, k):
        #Write Code Here
        res = 0
        
        for i in range(k):
            if arr[i] <= arr[k]:
                res += 1
                
        for i in range(k+1, len(arr)):
            if arr[i] < arr[k]:
                res += 1
                
        return res

File: 24. Stable Sort and Position/main.cpp
//Back-end complete function template in C++

class Solution {

  public:
    // Function to get the index of an element in a sorted array.
    int getIndexInSortedArray(vector<int>& arr, int k) {
        // Write Code Here
        int res = 0;
        int n = arr.size();
        
        for (int i = 0; i < k; i++)
            if (arr[i] <= arr[k])
                res += 1;
                
        for (int i = k + 1; i < n; i++)
            if (arr[i] < arr[k])
                res += 1;
                
        return res;
    }
};